iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/mempool_introspection.hpp

```diff
@@ -52,7 +53,7 @@ class MemPoolIntrospection
         TERMINATE
     };
 
-    using Topic = MemPoolIntrospectionTopic;
+    using Topic = MemPoolIntrospectionInfoContainer;
```

Comment:
`using Topic` is wrong terminology now.
How about `using InfoContainer` ?

Reply:
No comment to this ? I think calling something a topic that isn't a topic a bit confusing.